generate_schedule.py

```diff
@@ -117,18 +130,18 @@ def clamp_to_gc(power):
             vehicle_flex = power_needed = v2g_flex = 0
         vehicles_present = num_cars_present > 0
 
-        battery_flex = bat_init_discharge_power if step_i == 0 else bat_full_discharge_power
+        bat_flex_discharge = bat_init_discharge_power if step_i == 0 else bat_full_discharge_power
+        battery_flex_charge = bat_flex_discharge / flex["batteries"]["efficiency"]
```

Comment:
Good point. I think however that dividing by efficiency is not called for here since flex["batteries"]["power"] already represents the max charging power. 
Before, dividing bat_flex_discharge by efficiency was necessary to get back an approximation of what is stored in flex["batteries"]["power"] since bat_flex_discharge = sum([b.max_power * b.efficiency for b in batteries]).